modules/testing/integration-tests/README.md

```diff
@@ -0,0 +1,121 @@
+# Integration Tests Module
+
+## Description
+
+This module creates infrastructure to run integration tests of various `seed-farmer` manifest(s) that are managed in a repository. Specifically, an AWS CodePipeline is provisioned that runs `seedfarmer apply` & `seedfarmer destroy` in sequence for any one or multiple manifests that are passed to the module. The pipeline is connected to a Github repository via an OAuth Token.
+
+## Usage 
+### Prerequisites
+1. Github Personal Access Token with the following permissions to your private repository. ![Token Permissions](./images/github_token_permissions.png)
+2. Store Token in Secrets Manager with type `Plaintext`. ![Secrets Manager](./images/secrets_manager.png)
+3. Reference Secret Value in Module Manifest: e.g.
+```yaml
+---
+name: integration-test
+path: modules/testing/integration-tests
+targetAccount: integ
+parameters:
+  - name: manifest-paths
+    value: manifests/example-integration-tests/deployment.yaml
+  - name: repo-owner
+    value: my-github-org-name
+  - name: repo-name
+    value: my-repo-name
+  - name: oauth-token-secret-name
+    value: my-oauth-token
+```
+
+### Deployment
+It is recommended that you do not deploy your integration test module in the same region/account you deploy the rest of your modules to. Use a different target region and/or account to avoid resource conflicts during invocations of the integration tests pipeline.
+
+**Account mapping alias must be one of `["integ","integration","integ-tests", "integration-tests",]` for the integration test to work properly**
+
+[Example deployment manifest using an alias for integration tests region](../../../manifests/example-integration-tests/deployment.yaml):
+
+```yaml
+name: example-integration-tests
+toolchainRegion: us-west-2
+forceDependencyRedeploy: True
+groups:
+  - name: optionals
+    path: manifests/example-integration-tests/optional-modules.yaml
```

Comment:
`optional-modules.yaml` file is missing in the manifests dir. Did you mean `core-modules.yaml` instead?